src/ResourceManagement/RecoveryServices.Backup/RecoveryServices.Tests/ScenarioTests/AzureSql/AzureSqlPolicyTests.cs

```diff
@@ -0,0 +1,181 @@
+﻿//
+// Copyright (c) Microsoft.  All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+using System;
+using System.Collections.Generic;
+using System.Configuration;
+using System.Linq;
+using System.Net;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+using Hyak.Common;
+using Microsoft.Azure.Management.RecoveryServices.Backup;
+using Microsoft.Azure.Management.RecoveryServices.Backup.Models;
+using Microsoft.Azure.Test;
+using RecoveryServices.Tests.Helpers;
+using Microsoft.Azure;
+
+namespace RecoveryServices.Tests
+{
+    public class AzureSqlPolicyTests : RecoveryServicesTestsBase
+    {
+        [Fact]
+        public void ListRecoveryServicesAzureSqlProtectionPolicyTest()
```

Comment:
Can we shorten this test name?  It impacts the path length of the recording file, which could cause problems depending on where you put your clone of the repo.